README.md

```diff
@@ -1 +1,52 @@
-# clojure-workshop
\ No newline at end of file
+# Clojure Workshop
+
+Basic workshop/course on Clojure programming.
+
+## Prerequisite
+
+##### Java
+Version 1.8.0 or higher
+The command: `java -version` should output: `[...] version "1.8.0_202"`
+##### Lein
+[Lein](https://leiningen.org/) Version 2.8.0 or higher
+The command: `lein -v` should output: `Leiningen 2.8.1 on Java 1.8.0_202 OpenJDK 64-Bit Server VM`
```

Comment:
`2.9.1` is the latest lein, or is there a reason why we would be using `2.8.1`?

Reply:
I thought this was the version Ykä had. We can change this, no problem. I myself have 2.9.0 :)